framework/web/helpers/CHtml.php

```diff
@@ -2197,6 +2197,10 @@ public static function activeRadioButtonList($model,$attribute,$data,$htmlOption
 	 * make the error summary to show only the first error message of each attribute.
 	 * If this is not set or is false, all error messages will be displayed.
 	 * This option has been available since version 1.1.3.
+	 * Another special option named 'encode' is recognized, which when set true, will
```

Comment:
It should be `false`:

```suggestion
	 * Another special option named 'encode' is recognized, which when set false, will
```

Reply:
Thanks, I've changed it and condition to correct